tests/integration_tests/build/test_coverage.py

```diff
@@ -32,6 +32,44 @@
 """Regex for extracting number of total executable lines found by kcov."""
 
 
+@pytest.mark.timeout(120)
+@pytest.mark.skipif(
+    platform.machine() != "x86_64",
+    reason="kcov hangs on aarch64"
```

Comment:
We should change the reason here because this test doesn't actually use in any way `kcov`.
```suggestion
    reason="no need to test it on multiple platforms"
```

Reply:
Done.